tests/test__greenness.py

```diff
@@ -4,8 +4,15 @@
 """
 import unittest
 
-from eventlet.green import urllib2, BaseHTTPServer
+from eventlet.green import BaseHTTPServer
 from eventlet import spawn, kill
+from eventlet.support import six
+
+if six.PY2:
+    from eventlet.green.urllib2 import HTTPError, urlopen
```

Comment:
How do you see it being imported as a module name when in the else block we have to do 2 imports from 2 different modules anyway?